nd4j/nd4j-backends/nd4j-tests/src/test/java/org/nd4j/autodiff/opvalidation/LayerOpValidation.java

```diff
@@ -1013,6 +1015,115 @@ public void testConv3dBasic() {
         assertNull(err);
     }
 
+    @Test
+    public void testLayerNorm() {
+        final INDArray random = Nd4j.rand(new int[]{10, 4});
+        final INDArray standardized = Nd4j.emptyLike(random);
```

Comment:
Hm... didn't realize we had an emptyLike... not a good method to use - it's just Nd4j.create but without inheriting datatype. You can use ```random.like()``` or ```random.ulike()``` instead.
https://github.com/deeplearning4j/deeplearning4j/issues/7252